docs/Installation/Installation_Guide.rst

```diff
@@ -1,579 +1,581 @@
+   | `Grafana`_              | Monitoring dashboards for cluster services              | http://localhost:8001/api/v1/namespaces/kube-ops/services/kube-prometheus-grafana:http/proxy/   |                                       |
+   +-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+---------------------------------------+
+   | `Cerebro`_              | An administration and monitoring console for            | http://localhost:8001/api/v1/namespaces/kube-ops/services/cerebro:http/proxy/                   | When accessing Cerebro, connect it to |
+   |                         | Elasticsearch clusters                                  |                                                                                                 | http://elasticsearch:9200 to operate  |
+   |                         |                                                         |                                                                                                 | the MetalK8s Elasticsearch cluster.   |
+   +-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+---------------------------------------+
+   | `Kibana`_               | A search console for logs indexed in Elasticsearch      | http://localhost:8001/api/v1/namespaces/kube-ops/services/http:kibana:/proxy/                   | When accessing Kibana for the first   |
+   |                         |                                                         |                                                                                                 | time, set up an *index pattern* for   |
+   |                         |                                                         |                                                                                                 | the ``logstash-*`` index, using the   |
+   |                         |                                                         |                                                                                                 | ``@timestamp`` field as *Time Filter  |
+   |                         |                                                         |                                                                                                 | field name*.                          |
+   +-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+---------------------------------------+
+
+See :doc:`../architecture/cluster-services` for more about these services

```

Comment:
Do these docs build?

Reply:
Removed bad docs link. 

Can't comment about build other than to note that the merge request went through, and the linter for RST is quite permissive/nonexistent.

Reply:
This would only be found by actually building the docs. For MetalK8s, we're working on a step in CI which performs this, and as such makes sure any proposed change to the docs doesn't break its build.